Relativity.Export.SDK.md

```diff
@@ -3,9 +3,31 @@
 [![nuget](https://img.shields.io/nuget/v/Relativity.Export.SDK.svg)](https://www.nuget.org/packages/Relativity.Export.SDK)
 
 The Relativity Export SDK package contains interfaces for interacting with Relativity Export APIs via .NET.
+## 3.2.7
+### Release Notes
+- Implemented `DateTimeFormat` to `LoadFileSettings` to allow for custom date time formats.
 
-## 3.2.1
+## 3.2.6

```

Comment:
1. We will be publishing version 3.2.7, so I would aggregate all release notes there.
2. We should add only release notes which are relevant for the external developers. In this case, that would be only "Implemented `DateTimeFormat` to `LoadFileSettings` to allow for custom date time formats.". Samples and analyzer are not part of the nuget package.

Reply:
You're right.
Done.